buildconfig/stubs/pygame/math.pyi

```diff
@@ -18,91 +18,195 @@ if sys.version_info >= (3, 9):
+    def __bool__(self) -> bool: ...
+    def __iadd__(self: _TVec, other: Union[Sequence[float], _TVec]) -> _TVec: ...
+    def __isub__(self: _TVec, other: Union[Sequence[float], _TVec]) -> _TVec: ...
+    @overload
+    def __imul__(self: _TVec, other: Union[Sequence[float], _TVec]) -> float: ...
+    @overload
+    def __imul__(self: _TVec, other: float) -> _TVec: ...
+    def __copy__(self: _TVec) -> _TVec: ...
+    copy = __copy__
+    def __safe_for_unpickling__(self) -> Literal[True]: ...
+    def __contains__(self, other: float) -> bool: ...  # type: ignore[override]
+    def dot(self: _TVec, other: Union[Sequence[float], _TVec]) -> float: ...
+    def cross(self: _TVec, other: Union[Sequence[float], _TVec]) -> _TVec: ...
+    def magnitude(self) -> float: ...
+    def magnitude_squared(self) -> float: ...
+    def length(self) -> float: ...
+    def length_squared(self) -> float: ...
+    def normalize(self: _TVec) -> _TVec: ...
+    def normalize_ip(self) -> None: ...
+    def is_normalized(self) -> bool: ...
+    def scale_to_length(self, value: float) -> None: ...
+    def reflect(self: _TVec, other: Union[Sequence[float], _TVec]) -> _TVec: ...
+    def reflect_ip(self: _TVec, other: Union[Sequence[float], _TVec]) -> None: ...
+    def distance_to(self: _TVec, other: Union[Sequence[float], _TVec]) -> float: ...
+    def distance_squared_to(
+        self: _TVec, other: Union[Sequence[float], _TVec]
+    ) -> float: ...
+    def lerp(
+        self: _TVec,
+        other: Union[Sequence[float], _TVec],
+        value: float,
+    ) -> _TVec: ...
+    def slerp(
+        self: _TVec,
+        other: Union[Sequence[float], _TVec],
+        value: float,
+    ) -> _TVec: ...
+    def elementwise(self: _TVec) -> VectorElementwiseProxy[_TVec]: ...
+    def rotate(
+        self: _TVec, angle: float, axis: Union[Sequence[float], _TVec]
+    ) -> _TVec: ...
+    def rotate_rad(
+        self: _TVec, angle: float, axis: Union[Sequence[float], _TVec]
+    ) -> _TVec: ...
+    def rotate_ip(
+        self: _TVec, angle: float, axis: Union[Sequence[float], _TVec]
+    ) -> None: ...
+    def rotate_rad_ip(
+        self: _TVec, angle: float, axis: Union[Sequence[float], _TVec]
+    ) -> None: ...
+    def rotate_ip_rad(
+        self: _TVec, angle: float, axis: Union[Sequence[float], _TVec]
+    ) -> None: ...
+    def angle_to(self: _TVec, other: Union[Sequence[float], _TVec]) -> float: ...
+    def move_towards(
+        self: _TVec,
+        target: Union[Sequence[float], _TVec],
+        max_distance: float,
+    ) -> _TVec: ...
+    def move_towards_ip(
+        self: _TVec,
+        target: Union[Sequence[float], _TVec],
+        max_distance: float,
+    ) -> None: ...
+    @overload
+    def clamp_magnitude(self: _TVec, max_length: float) -> _TVec: ...
+    @overload
+    def clamp_magnitude(self: _TVec, min_length: float, max_length: float) -> _TVec: ...
+    @overload
+    def clamp_magnitude_ip(self, min_length: float) -> None: ...
```

Comment:
This is supposed to be `clamp_magnitude_ip(self, max_length)`, it seems like there was a mistake with typing from the clamp_magnitude PR. Might as well fix it here.

Reply:
good catch!